CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - `SEARCH_ERROR`
     - `SEARCH_REQUEST`
     - `SEARCH_RESPONSE`
+    - `TRACKER_SEARCH`
```

Comment:
This package has now been released, so you should add a new `## [Unreleased]` section instead. Use the `scripts/add-unreleased-section.sh` script to help you. It will add in a template that you can fill out.